src/styles/settings/calendar.scss

```diff
@@ -0,0 +1,58 @@
+$day-size: 100px;
+$calendar-padding: 82px;
+$calendar-size: ($day-size * 7) + (2 * $calendar-padding) + 6;
+$bg-color: #eee;
+$days: (
+  mon: 0,
+  tue: $day-size + 1,
+  wed: 2 * $day-size + 2,
+  thu: 3 * $day-size + 3,
+  fri: 4 * $day-size + 4,
+  sat: 5 * $day-size + 5,
+  sun: 6 * $day-size + 6,
+);
+
+.calendar {
+  margin-left: 77px;
```

Comment:
Consider using a variable for `margin-left` to maintain consistency and make future adjustments easier.